src/sources/util/mod.rs

```diff
@@ -12,6 +12,7 @@ mod encoding_config;
 pub mod finalizer;
 #[cfg(all(unix, feature = "sources-dnstap"))]
 pub mod framestream;
+pub mod grpc;
```

Comment:
Looks like CI highlighted the fact that this gets pulled in even when nothing is using it, which results in errors.

You can pull it in conditionally pretty simply:
```suggestion
#[cfg(feature = "sources-vector")]
pub mod grpc;

Reply:
Done